packaging/Dockerfile.leap.42.3

```diff
@@ -32,8 +32,6 @@ RUN if ! grep "^#includedir /etc/sudoers.d" /etc/sudoers; then              \
     fi;                                                                     \
     echo "build ALL=(ALL) NOPASSWD: /usr/bin/build" > /etc/sudoers.d/build; \
     echo "build ALL=(ALL) NOPASSWD: /bin/rpm" >> /etc/sudoers.d/build;      \
```

Comment:
Why is `/bin/rpm` needed on Leap 42.3 but not Leap 15?

Reply:
The Leap-15 dockerfile is not used for this.  The packaging repository will need to be updating before this is changed here.

The /bin/rpm is used for installing a GPG key needed for a repository only used for SLES 12.3 and Leap 42.3, but the makefile_packaging.mk is downloading and installing that key for all SLES-15/LEAP-15 also.

More likely the makefile_packaging.mk in the packaging repository should be fixed to not install the key when it is not needed.